lib/cookie.js

```diff
@@ -69,11 +83,13 @@ export default class Cookie {
     return result;
   }
 
+  
   // returns a cookie that expires on 01 Jan 1970 
   // assign the returned cookie to an existing cookie to delete that cookie
-  removeCookie (key, options) {
-    return this.writeCookie(key, '', _.extend({}, options, {
+  function expireCookie (key, options) {
+    return createJSCookie(key, '', _.extend({}, options, {
       expires: 'Thu, 01 Jan 1970 00:00:00 GMT'
     })); 
   }
-}
\ No newline at end of file
+
+  module.exports = {createJSCookie, createJWPCookie, getValue, expireCookie};
```

Comment:
instead of `module.exports = `, in ES6 you should just do `export`

and we put a space inside export/import braces, so I'd say:

```js
export { createJSCookie, createJWPCookie, getValue, expireCookie };
```